doc/guide/transactions-and-threading.rst

```diff
@@ -0,0 +1,394 @@
+============================
+Transactions and concurrency
+============================
+
+.. contents::
+
+`Transactions <https://en.wikipedia.org/wiki/Database_transaction>`_
+are a core feature of ZODB.  Much has been written about transactions,
```

Comment:
Well, we aren't referring to the package here.

Did anything else ever use the transaction package?